### PR TITLE
http request node respects now environment vars

### PR DESCRIPTION
**Pull request by klausfmh**
required to allow accessing extranet sides if an http proxy is required

### CODE REVIEW

**Review by Zluurk — APPROVED** [review]

File: pypeman/contrib/http.py [hunk 1]
```diff
@@ -302,7 +302,11 @@ async def handle_request(self, msg):
         data = None
         if method.lower() in ['put', 'post', 'patch']:
             data = msg.payload
-        async with aiohttp.ClientSession(connector=conn, cookies=cookies) as session:
+        async with aiohttp.ClientSession(
+            connector=conn,
+            cookies=cookies,
+            trust_env=True,  # respect e.g. http_proxy env vars
```
Comment (mrch81) [inline pypeman/contrib/http.py:308]: After the discussion yesterday I am wondering whether we should add the version of aiohttp in the requirements. (trust_env was introduced in 2.3. So no issues for this particular change)

**Review by mrch81 — APPROVED** [review]

File: pypeman/contrib/http.py (hunk 1 above)
Comment (klausfmh) [reply pypeman/contrib/http.py:308]: might add >=2.3 to requirements just to be sure.

Comment (klausfmh) [reply pypeman/contrib/http.py:308]: 2.3 dates from 2017. so better to keep requirements simple instead of adding this constraint

**Review by asixela — APPROVED** [review]

File: pypeman/contrib/http.py (hunk 1 above)
Comment (mrch81) [reply pypeman/contrib/http.py:308]: > 2.3 dates from 2017. so better to keep requirements simple instead of adding this constraint

Makes sense, thanks.

**Review by quentinql — APPROVED** [review]

**Pull request merged**